### DISCUSSION TITLE
Data Seq Error

### DISCUSSION THREAD

**Discussion by Kipio37** [picocomputer/community#22] [Q&A] [answered]
I have a newly self-assembled system, and VGA is displaying just fine.  When I start the computer with just a USB hub connected to RIA, the screen looks normal.  However, when I start with a USB keyboard plugged into the hub, I get this error:

*** PANIC ***
Data Seq Error

The same thing happens if I hot plug the USB keyboard into the hub.  I've tried two different USB keyboards, with the same results.  Unfortunately I only have one hub with microUSB.  It does claim to be OTG-comptable, though.

I'm not sure how to troubleshoot but I would be happy to do so.  I can probably set up serial if that would help, although I haven't determined yet which header pins are UART.  (I assume those are exposed in the GPIO or PIX headers.)

**Comment by rumbledethumps** [comment]
The Pico VGA is also a USB-to-UART and ARM debugger for the RIA if you hooked up those optional 3 SWD pins.

I did a web search for "TinyUSB Data Seq Error" and found this is a known issue: https://github.com/hathach/tinyusb/issues/1997

**Comment by Kipio37** [comment, marked as answer]
I bought a microUSB to USB-A adapter and plugged my keyboard directly into it, and no more error.  Previously I had plugged into the RIA a microUSB to USB-A hub, into which I plugged my keyboard.  Instead, I've tried plugging a USB-A hub into my microUSB adapter, and am no longer receiving an error with this configuration either.  So I guess my suggestion to anyone running into this issue might be to try a different USB hub.
